tesseract_motion_planners/descartes/include/tesseract_motion_planners/descartes/descartes_robot_sampler.h

```diff
@@ -66,6 +66,8 @@ class DescartesRobotSampler : public descartes_light::WaypointSampler<FloatType>
 
   std::vector<descartes_light::StateSample<FloatType>> sample() const override;
 
+  virtual std::ostream& format(std::ostream& out) const override;
```

Comment:
Looks like this references a change in descartes as well. Do you have that branch?

Reply:
nevermind.. found it